application/src/main/java/org/opentripplanner/routing/api/request/RouteRequest.java

```diff
@@ -296,6 +297,17 @@ public List<ViaLocation> getViaLocations() {
     return via;
   }
 
+  /**
+   * Return the via locations that are of type {@link VisitViaLocation}.
+   */
+  public List<VisitViaLocation> getVisitViaLocations() {
```

Comment:
I know this class is not consistent, but this method is not according to the naming conventions.